packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -2879,6 +2879,12 @@ function commitMutationEffectsOnFiber(
       // TODO: Add explicit effect flag to set _current.
       finishedWork.stateNode._current = finishedWork;
 
+      // Offscreen stores pending changes to visibility in `_pendingVisibility`. This is
+      // to support batching of `attach` and `detach` calls.
+      finishedWork.stateNode._visibility &= ~OffscreenDetached;
```

Comment:
Nit: assign the instance to a variable, since `stateNode` is any typed.

```js
const offscreenInstance: OffscreenInstance = finishedWork.stateNode;
```